server/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "gitkv"
+version = "0.1.0"
+description = "gitkv is a server for using git as a key value store for text files"
+authors = ["Intent HQ <engineering@intenthq.com>"]
+edition = "2018"
+
+[dependencies]
+git = { path = "../git" }
+actix-web = "0.7.*"
+clap = "2"
+git2 = "0.8.*"
+listenfd = "0.3"
+serde = "1"
```

Comment:
Make this a full range, `1.0.89` — otherwise it will allow potentially breaking changes to major versions as upgrades permitted by a `cargo update`.